test/hotspot/jtreg/compiler/vectorization/TestPopulateIndex.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+* @test
+* @summary Test vectorization of loop induction variable usage in the loop
+* @requires vm.compiler2.enabled
+* @requires vm.cpu.features ~= ".*avx2.*"
+* @requires os.arch=="amd64" | os.arch=="x86_64"
```

Comment:
Can we simplify `os.arch=="amd64" | os.arch=="x86_64"` to `os.simpleArch == "x64"` ?

This test runs on x86 only. It would be nice if it can run on AArch64 as well. So perhaps something like
```
 28 * @requires (os.simpleArch == "x64" & vm.cpu.features ~= ".*avx2.*") |
 29 *           (os.simpleArch == "aarch64" & vm.cpu.features ~= ".*sve.*")
```

Reply:
@pfustc  Yes, changed the requires to as suggested by you.